examples/prismic/README.md

```diff
@@ -0,0 +1,171 @@
+
+This integration allows you to combine the power of Prismic's content management with Makeswift's visual page builder. Create dynamic, content-rich websites with a seamless development experience.
+
+For a detailed guide on setting up and using this integration, visit:
+https://www.makeswift.com/integrations/Prismic
+
+## Prerequisites
+
+- Node.js 18.x or later
+- Makeswift account (for visual page building)
+- Prismic account (for content management)
+- Basic familiarity with Next.js App Router
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
+## Project Structure
+
+```
+Prismic/
+├── app/                   # Next.js App Router pages
+│   ├── blog/              # Blog-related routes
+│   ├── api/               # API endpoints
+│   └── [[...path]]/       # Dynamic Makeswift routes
+├── components/            # Reusable React components
+│   └── Prismic/        # Prismic-specific components
+│       ├── common/        # Common scalar components
+│       ├── entries/       # Entry-specific components
+│       │   └── BlogPost/  # BlogPost entry components
+│       └── queries/       # GraphQL queries
+├── lib/                   # Utility functions and configs
+└── generated/             # Auto-generated types from Prismic
+```
+
+## Quick Start
+
+### Option 1: Deploy to Vercel
+
+The deploy link below includes integrations with Prismic and Makeswift:
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2FPrismic&project-name=Prismic-makeswift-example&repository-name=Prismic-makeswift-example&redirect-url=https%3A%2F%2Fapp.makeswift.com&integration-ids=oac_51ryd7Pob5ZsyTFzNzVvpsGq,oac_9z8CliUmYcKI8qoY0JtmSyzz&external-id=Prismic-makeswift)
+
+### Option 2: Local Development
+
+1. Set up the project:
+
+   ```bash
+   npx makeswift@latest init --example=Prismic
+   ```
+
+2. Install dependencies:
+
+   ```bash
+   pnpm install
+   ```
+
+3. Configure environment variables in `.env.local`:
+
+   ```
+   PRISMIC_REPOSITORY_NAME=your_respository_name
+   NEXT_PUBLIC_PRISMIC_API_TOKEN=your_access_token
+   MAKESWIFT_SITE_API_KEY=your_makeswift_api_key
+   ```
+
+4. Setting up Prismic
+   To get started with blog posts, you'll need to set up a content model in Prismic. Here's what we recommend:
+
+Create a new content type called "BlogPost" with the following fields:
+
+- **Title | Short text [Entry title]** - A short text field for the post title
+- **Feed Date | Date & time** - When the post should appear in feed
+- **Body | Rich text** - The main content as rich text
+- **Hero | Media type** - An optional hero image for the post
+
+Be sure to add a few blog posts for testing purposes, and connect them to an Author
+
+4. Generate Prismic types:
+
+   ```bash
+   pnpm codegen-ts
+   ```
```

Comment:
I followed the instructions above, and I'm getting these errors on the codegen step:

<img width="1209" alt="codegen errors" src="https://github.com/user-attachments/assets/37d7ccf1-da64-4771-b477-bbd4c4309e9b" />